akka-actor-typed/src/main/scala/akka/actor/typed/internal/routing/PoolRouterImpl.scala

```diff
@@ -30,6 +30,12 @@ private[akka] final case class PoolRouterBuilder[T](
 
   def withRoundRobinRouting(): PoolRouterBuilder[T] = copy(logicFactory = () => new RoutingLogics.RoundRobinLogic[T])
 
+  def withConsistentHashingRouting(
+    virtualNodesFactor: Int = 0,
+    mapping: ConsistentHashMapping[T] = ConsistentHashingLogic.emptyHashMapping,
+    system: ActorSystem[T]): PoolRouterBuilder[T] =
+    copy(logicFactory = () => new RoutingLogics.ConsistentHashingLogic[T](virtualNodesFactor, mapping, system))
```

Comment:
Default parameters are really tricky to maintain binary compatible, so we need to avoid those, I don't think these makes sense anyway.